src/common/components/theme-family-customizer.tsx

```diff
@@ -1,47 +1,32 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { ThemeProvider, PartialTheme } from '@fluentui/react';
 import { NamedFC } from 'common/react/named-fc';
-import { DefaultThemePalette } from 'common/styles/default-theme-palette';
-import { FastPassThemePalette } from 'common/styles/fast-pass-theme-palette';
-import { HighContrastThemePalette } from 'common/styles/high-contrast-theme-palette';
+import { DefaultTheme } from 'common/styles/default-theme';
+import { FastPassTheme } from 'common/styles/fast-pass-theme';
+import { HighContrastTheme } from 'common/styles/high-contrast-theme';
 import { UserConfigurationStoreData } from 'common/types/store-data/user-configuration-store';
-import { createTheme, Customizer, ITheme } from 'office-ui-fabric-react';
 import * as React from 'react';
 
 export type ThemeFamily = 'default' | 'fast-pass';
 
-// createTheme is somewhat expensive; don't call it in render()
-const themeFamilyDefaultThemes: { [themeFamily in ThemeFamily]: ITheme } = {
-    default: createTheme(DefaultThemePalette),
-    'fast-pass': createTheme(FastPassThemePalette),
+const themeFamilyDefaultThemes: { [themeFamily in ThemeFamily]: PartialTheme } = {
+    default: DefaultTheme,
+    'fast-pass': FastPassTheme,
 };
-const highContrastTheme = createTheme(HighContrastThemePalette);
 
 export type ThemeFamilyCustomizerProps = {
     userConfigurationStoreData: UserConfigurationStoreData;
     themeFamily: ThemeFamily;
     children: React.ReactNode;
 };
 
-// This is a store-state-aware version of the Fluent UI <ThemeProvider> component. It's intended
```

Comment:
Was removing this comment intentional? The one that was on 38-44 is obsolete, but I don't think this one is

Reply:
True, I'll put it back. Good catch, thank you